server/routes/actions/actions_routes.py

```
@@ -144,21 +151,22 @@ def list_actions(user_id):
-                        except Exception:
-                            logger.debug("Failed to lazy-seed system actions")
+                expected_keys = [a["system_key"] for a in SYSTEM_ACTIONS]
+                cur.execute(
+                    "SELECT system_key FROM actions WHERE org_id = %s AND is_system = true AND system_key = ANY(%s)",
+                    (org_id, expected_keys),
+                )
+                existing_keys = {row[0] for row in cur.fetchall()}
+                if len(existing_keys) < len(expected_keys):
+                    try:
+                        seed_system_actions(org_id, user_id)
+                    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)